README.md

```diff
@@ -10,10 +10,10 @@ Search for `Capitalize` in the _Extensions_ section.
 
 ## Usage
 
-Select the text you want to capitalize and execute the `Capitalize title` command via the command palette.
+Select the text you want to capitalize and execute the `Capitalize Title` command via the command palette.
 
 ## Configuration
 
-#### `capitalize.special`
+**`capitalize.special`**
```

Comment:
```suggestion
#### `capitalize.special`
```

Even though there's only one configuration option now, I'd rather have them as headings to be able link to them directly.

Reply:
Okay, that makes sense :+1: